### PR TITLE
Upgrade to Ember 2.11 and address the moving of ember away from bower

### PR DESCRIPTION
**Pull request by mfeckie**
I did some poking around too and discovered the answer

https://github.com/ember-cli/ember-cli/issues/6712

We're using 2.11, so I need this, though I note you said that you're maintaining 2.10.

This PR, therefore, may not yet be useful for you, but should help when you come to make the transition.

### CODE REVIEW

**Comment by evrowe** [comment]
Ah, nice find. We'll keep this around for when we're ready to cut a 2.11 compatible release.

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1,8 @@
-# See http://help.github.com/ignore-files/ for more about ignoring files.
+# See https://help.github.com/ignore-files/ for more about ignoring files.
 
 # compiled output
 /dist
 /tmp
-/tests/dummy/public/docs
-vendor/feature-flags.js
-vendor/ember-fountainhead.css
```
Comment (evrowe) [inline .gitignore]: These three lines need to remain in the `.gitignore` list

**Comment by DHedgecock** [comment]
@mfeckie Thanks for finding that solution, it ended up being simpler than the one I had originally.

I just released v2.3 which uses Ember 2.11 and the import pattern you posted.

**Comment by mfeckie** [comment]
Thanks

**Pull request closed without merging**